### PR TITLE
 request the picture and name from animal API

### PR DESCRIPTION
**Pull request by jbundy611**
added code for animal api to proper file

### CODE REVIEW

**Review by Stewtec612 — COMMENTED** [review]
Working well. we should wait till we are sure the animal picture will open inside the webpage before we merge

**Comment by claraj** [comment]
If the code is returning a valid URL - like, if you paste it in a browser you see the correct animal picture -  I think you can assume that it will display correctly in the webpage.

File: animal_api.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import requests
+#opens channel to animal API
+animal_url = "https://zoo-animal-api.herokuapp.com/animals/rand"
+
+response = requests.get(animal_url)
+
```
Comment (FireL0ki) [inline animal_api.py:6]: nit: delete extra line?

**Review by FireL0ki — APPROVED** [review]
This looks good-- good basic structure, clear function names.
Based on looking at the API, this looks to me like it should function as expected.

We will want to add error handling to this.

**Pull request merged**